docs/reference/queues.md

```diff
@@ -63,26 +63,26 @@ traffic to the destination domain.
 
 ```lua
 kumo.on('init', function()
-  kumo.define_egress_source {
+  kumo.make_egress_source {
```

Comment:
not your fault: this whole section needs to be updated because `make_egress_source` is only really valid when returned up via the `get_egress_source` event handler.

The userguide docs over in https://docs.kumomta.com/userguide/configuration/sendingips/ are closer to the current state of the world

Reply:
I changed the event from `init` to `make_egress_source`, but not sure if that's the only change that's needed? Maybe the description above this change needs to be updated as well? Let me know.